my-app/src/app/core/ui/header/breadcrumbs/breadcrumbs.component.html

```diff
@@ -1,3 +1,9 @@
-<div class="breadcrumbs layout-padding vertical-align--center">
-  Courses
+<div *ngIf="breadcrumbsService.notEmptyBreadcrumbs?.length" class="breadcrumbs layout-padding vertical-align--center">
+  <ng-container *ngFor="let breadcrumb of breadcrumbsService.getLinkBreadCrumbs()">
+    <span *ngIf="breadcrumb.label">
+      <a [routerLink]="breadcrumb.url" class="breadcrumbs__item">{{breadcrumb.label}}</a>
+      &nbsp;/&nbsp;
+    </span>
+  </ng-container>
+  <span class="breadcrumbs__item">{{breadcrumbsService.getLast().label}}</span>
```

Comment:
you can use `ngFor` variables to check if the item is the last one, see https://angular.io/api/common/NgForOf#local-variables